common/djangoapps/philu_commands/management/commands/fetch_course_structures.py

```diff
@@ -0,0 +1,47 @@
+"""
+Django management command to fetch course structures for given course ids
+"""
+import json
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+from philu_commands.helpers import generate_course_structure
+
+log = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """
+    This command returns the course structure for all the course ids given in arguments
```

Comment:
I think 'This command prints' or `This command displays' would be more appropriate.